sdks/python/apache_beam/io/gcp/bigquery_file_loads_test.py

```diff
@@ -83,6 +84,23 @@
 _ELEMENTS = list([json.loads(elm[1]) for elm in _DESTINATION_ELEMENT_PAIRS])
 
 
+class CustomRowCoder(coders.Coder):
+  """
+  Custom row coder that also expects strings as input data when encoding
+  """
+
+  def __init__(self):
+    self.coder = bigquery_tools.RowAsDictJsonCoder()
+
+  def encode(self, table_row):
+    if type(table_row) == str:
+      table_row = json.loads(table_row)
+    return self.coder.encode(table_row)
+
+  def decode(self, encoded_table_row):
+    return self.coder.decode(encoded_table_row)
+
+
```

Comment:
We can merge this now. I am just curious: Why did you add this custom coder?

Reply:
Please take a look at https://github.com/apache/beam/pull/8621#discussion_r287935702 - and let us know if you think we should do something else here.